Explorer/Assets/DCL/Character/CharacterCamera/Systems/ControlCinemachineVirtualCameraSystem.cs

```diff
@@ -95,70 +95,73 @@ private void SetActiveCamera(ICinemachinePreset cinemachinePreset, in CameraComp
                     // Disable Player input
                     inputMapComponent.Active &= ~InputMapComponent.Kind.Player;
 
+                    SetDefaultFreeCameraPosition(in cinemachinePreset);
+
                     break;
             }
         }
 
         [Query]
         private void HandleZooming(ref CameraComponent cameraComponent, ref CameraInput input, ref ICinemachinePreset cinemachinePreset, ref CinemachineCameraState state)
         {
-            // Zoom out
-            if (input.ZoomOut)
+            if (cameraComponent.CameraInputChangeEnabled)
             {
-                switch (cameraComponent.Mode)
+                if (input.ZoomOut)
                 {
-                    // if we switch from FP to TP just zoom at 0 position
-                    case CameraMode.FirstPerson:
-                        SwitchCamera(CameraMode.ThirdPerson, ref cinemachinePreset, ref cameraComponent, ref state);
-
-                        // Reset zoom value
-                        state.ThirdPersonZoomValue = 0f;
```

Comment:
Since this PR already contains a lot of changes I prefer leaving that original code as it is right now and in any case change it in a different independent PR please.